### PR TITLE
fix documentation scripts

### PR DESCRIPTION
**Pull request by ofrimasad**

### CODE REVIEW

File: documentation/source/conf.py [hunk 1]
```diff
@@ -12,21 +12,25 @@
...
-project = 'SuperGradients'
-copyright = '2021, SuperGradients team'
-author = 'SuperGradients team'
+project = "SuperGradients"
+copyright = "2021, SuperGradients team"
+author = "SuperGradients team"
 
 # The full version, including alpha/beta/rc tags
-release = '1.0'
+release = "1.0"
```
Comment (shaydeci) [inline documentation/source/conf.py:32]: Change to 3.0.2 ?

File: documentation/source/sphinx-requirements.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
-sphinx
-sphinxcontrib-napoleon
-myst-parser
\ No newline at end of file
+Sphinx==5.3.0
+sphinxcontrib.napoleon
```
Comment (shaydeci) [inline documentation/source/sphinx-requirements.txt:2]: on purpose ?

**Review by shaydeci — CHANGES_REQUESTED** [review]
2 small comments inline.

File: documentation/source/sphinx-requirements.txt (hunk 2 above)
Comment (ofrimasad) [reply documentation/source/sphinx-requirements.txt:2]: yes. the previous version was very old

File: documentation/source/conf.py (hunk 1 above)
Comment (ofrimasad) [reply documentation/source/conf.py]: fixed in [e3a13fc](https://github.com/Deci-AI/super-gradients/pull/462/commits/e3a13fc0bc2ff7ba0dbc34c64291693e99109559)

**Review by shaydeci — APPROVED** [review]
LGTM

**Pull request merged**